### ISSUE TITLE
Config auditor will complain about missing npm scripts that we should probably consider optional

### ISSUE THREAD

**Issue by kethinov** [rooseveltframework/roosevelt#237]
e.g. remove the `eslint`, `stylelint`, or `lint` scripts, and the config auditor will complain that they are missing.

Strictly speaking, those (and possibly other things?) should be considered optional.

There is also probably a class of optional *but recommended* things too. I can't think of any right now, but in such cases, I would degrade this error to a warning, and explicitly note that roosevelt recommends that it be present even though it is optional.

Basically two extra layers: a class of optional configs that roosevelt shows a warning for when it's missing, and a class of optional configs that roosevelt will ignore when it's missing without showing an error or a warning.

The whole config auditor needs to be audited to identify configs which belong on these reduced levels of urgency.
